doc-content/shared-kie-docs/src/main/asciidoc/Workbench/RemoteAPI/security-management-rest-api-requests-proc.adoc

```diff
@@ -75,20 +76,20 @@ For curl utility:
 
 * `-u`: Enter the user name and password of the {CENTRAL} user with the `admin` role.
 * `-H`: Set the following headers:
-** `accept`: `application/json`
-** `content-type`: `application/json`
+** `Accept`: `application/json`
+** `Content-type`: `application/json`
```

Comment:
`Content-Type` would be even better as an alignment with Rest client option :)

Reply:
Done